RedHat-UX/red-hat-design-system
# fix(footer)!: rename global-footer to footer-universal

Closes #785 

## What I did

1. Rename `<rh-global-footer>` to `<rh-footer-universal>`
1. Alias `<rh-footer-universal>` to `<rh-global-footer>`
2. Update docs, demos, and examples


## Testing Instructions

1. Check that the rh-footer-universal demo works as expected
1. Run spandx and check that the `<rh-global-footer>` alias still works

## Notes to Reviewers
